bodhi/server/models.py

```diff
@@ -3008,6 +3008,10 @@ def comment(self, session, text, karma=0, author=None, anonymous=False,
         if not author:
             raise ValueError('You must provide a comment author')
 
+        if (not text and not karma and not karma_critpath and not
+                bug_feedback and not testcase_feedback):
+            raise ValueError('You must provide either some text or feedback')
```

Comment:
Hmm, strangely I found that if you try to enter an empty comment (with no bug feedback) on an Update that is associated with a Bug (for example, http://localhost:6543/updates/FEDORA-2018-576bea0c62) the ```bug_feedback``` here is actually not "falsey" like I would expect, so this exception is not raised.

The update in question already has bug feedback from another user (anonymous) and that appears to be the bug feedback being provided here.

However, this oddity is not due to the change you are making here so I think it does not block this pull request.

I've filed an issue about this at https://github.com/fedora-infra/bodhi/issues/2623.